boudicca.base/publisher-event-html/build.gradle.kts

```diff
@@ -5,4 +5,11 @@ plugins {
 dependencies {
     api("com.github.jknack:handlebars:4.4.0")
     api(project(":boudicca.base:search-client"))
+
+    api("com.microsoft.playwright:playwright:1.42.0")
```

Comment:
those should be testImplementation dependencies

Reply:
done.